client.go

```diff
@@ -51,7 +51,7 @@ type Client struct {
 	// TessdataPrefix can indicate directory path to `tessdata`.
 	// It is set `/usr/local/share/tessdata/` or something like that, as default.
 	// TODO: Implement and test
-	TessdataPrefix *string
+	TessdataPrefix string
```

Comment:
Since there is no real use case of empty prefix, there is no reason to distinguish "empty" value and "never set" value, that's why I have omit pointer. If prefix is empty, then in `client.init` function 
```go
var tessdataPrefix *C.char
if client.TessdataPrefix != "" {
	tessdataPrefix = C.CString(client.TessdataPrefix)
}
defer C.free(unsafe.Pointer(tessdataPrefix))
```

`tessdataPrefix` will remain uninitialized (nil pointer) and `nil` will be passed to C++ `Init` function as you wish.